Solution1/BusinessLayer/Command/BestWeatherCommand.cs

```diff
@@ -0,0 +1,28 @@
+﻿using BusinessLayer.Command.Abstract;
+using BusinessLayer.DTOs;
+using BusinessLayer.Services.Abstract;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BusinessLayer.Command
+{
+    public class BestWeatherCommand : ICommand<Dictionary<bool, List<WeatherResponseDTO>>>
+    {
+        private readonly IWeatherServiсe _weatherServiсe;
+        private readonly string _arrayCityNames;
+
+        public BestWeatherCommand(IWeatherServiсe weatherServiсe, string arrayCityNames)
```

Comment:
arrayCityNames string in console app, but what about if we use command in web api?

Reply:
+